projects/ngx-auth-firebaseui/src/lib/components/ngx-auth-firebaseui/auth.component.ts

```diff
@@ -245,10 +256,12 @@ export class AuthComponent implements OnInit, AfterViewInit, OnChanges, OnDestro
   async signOut() {
     try {
       this.isLoading = true;
+      this.loading.emit(true);
       this.changeDetectorRef.markForCheck();
       await this.authProcess.signOut();
     } finally {
       this.isLoading = false;
+      this.loading.emit(true);
```

Comment:
it should be `this.loading.emit(false)`

or not ?

Reply:
true. That's a mistake. Done